docs(competition): Competition clarification (#1239)
keephq/keep: docs/overview/comparison.mdx

Sometimes, it's easier to understand a tool's capabilities by comparing it to other tools in the same ecosystem. In this section, we'll discuss how Keep interacts with and compares to other tools in the ecosystem.

> ## Keep vs AIOps inside observability platforms (Splunk ITSI, Elastic AIOps, etc.)

Reviewer: How about:
```suggestion
## Keep vs observability traditional AIOps
```
It feels a little to much for a title?

Keep differs because it's able to correlate alerts between different observability platforms.

|                                       | Keep                                                           | Alternative                  |
| ------------------------------------- | -------------------------------------------------------------- | ---------------------------- |
| Aggregation of alerts from one platform | ✅                                                           | ✅                            |
| .. from multiple platforms            | ✅                                                             | ❌                            |
| Correlation of alerts between multiple sources | ✅                                                    | ❌                            |
| Alerts enrichment                     | ✅                                                             | ❌                            |
| Open source                           | ✅                                                             | ❌                            |
| Workflow automation                   | ✅                                                             | ❌                            |

> ## Keep vs dedicated AIOps solution (BigPanda, MoogSoft, etc.)

Reviewer: Same as other comment but maybe repeating BigPanda and Moogsoft twice is redundant?
```suggestion
## Keep vs traditional AIOps platforms
```

> BigPanda and Moogsoft are our direct competitors. Keep differs because it's open source, which makes integration and piloting easy. Our customer also noticed a significant difference in the alert correlation approach since we at Keep focus less on manual configuration and more on applying state-of-the-art AI models, meaning it's easier for Keep to correlate alerts, which never happened before.

Reviewer: Feels a little bit "testimonial"ish, is it what you meant? tried to apply some changes, but not 100% happy with my changes too 
```suggestion
Keep is an alternative to platforms like BigPanda and Moogsoft. 
We differ with many things but farmost by being open source, which makes integration and piloting easy. 
Customers who tried both traditional platforms and Keep note a significant difference in the alert correlation approach since Keep is a lot less manual and uses state-of-the-art AI models, meaning it's easier for Keep to correlate alerts, which never happened before.
```
